ui/apps/platform/src/Containers/Policies/Wizard/Step3/TableModal.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import {
+    Button,
+    ButtonVariant,
+    Modal,
+    ModalBoxBody,
+    ModalBoxFooter,
+    ModalVariant,
+    PageSection,
+    TextInput,
+} from '@patternfly/react-core';
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import isEqual from 'lodash/isEqual';
+import pluralize from 'pluralize';
+
+import LinkShim from 'Components/PatternFly/LinkShim';
+import useTableSelection from 'hooks/useTableSelection';
+import TableCellValue from 'Components/TableCellValue/TableCellValue';
+
+type TableModalProps = {
+    setValue: (value: any) => void;
+    value: any;
+    readOnly?: boolean;
+    rows: any;
+    columns: any;
+    typeText: string;
+};
```

Comment:
Does TypeScript compile with any of the following alternatives to `any`
* `setValue: (value: unknown) => void;`
* `value: unknown;`
* `rows: { id: string; link: string }[];`
* `columns: { Header: string; accessor: ((data: unknown) => string) | string }[];`